### DISCUSSION TITLE
ESP32 D0WD-V3 porting another ESP controller

### DISCUSSION THREAD

**Discussion by SanjayEIC** [Moddable-OpenSource/moddable#1283] [Porting]
Hi i am working ESP modules porting to another controller, ESP32 D0WD-V3  code is ready but i want to port on another controller (ESP controller) what is main important things' we easily port

**Comment by phoddie** [comment]
Just about all the ESP32 family of microcontrollers are [supported already](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/devices/esp32.md#platforms).
